public/css/about.css

```diff
@@ -234,10 +286,23 @@
 
 @media (max-width: 990px) {
   .socials .title {
+    padding-top: 0;
     text-align: center;
   }
 }
 
+@media (max-width: 990px) {
```

Comment:
These **three** media queries can be merged since they're all affecting the same section at the same screen-width:

```
@media (max-width: 990px) {
  .socials .title {
    padding-top: 0;
    text-align: center;
  }

 .socials .involve-about {
    padding-right: 0;
  }

  .socials {
    padding: 70px 55px;
  }
}
```